libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -102,6 +108,16 @@ class CppInterface : public testing::Test {
     return module.GetErrorMessage();
   }
 
+  // Compiles a shader, expects compilation success, and returns the output
+  // bytes.
+  std::string CompilationOutput(const std::string& shader,
```

Comment:
Ok.  Requireing the spvBinaryParse for the test adds a link dependency to shaderc that we don't want yet.
So keep the test as is.  Maybe add a TODO if you like saying how we could strengthen the test.